doc/tutorial/pairwise_alignment/index.md

```diff
@@ -209,7 +209,7 @@ the assignment. Et voilà, we have computed a pairwise alignment over aminoacid
 
 ## Alignment result
 
-So far we have only used the score, but obviously in many situations the final alignment is required, e.g. when
+So far we have only used the score, but obviously, in many situations the final alignment is required, e.g. when
```

Comment:
```suggestion
So far, we have only used the score. However, in many situations the final alignment is required, e.g. when
```

I think 2 sentences are ok; `however` seems fitting.
I'd definitely get rid of `obviously` :)